src/Testing.jl

```diff
@@ -0,0 +1,38 @@
+module Testing
+
+function get_bio_fmt_specimens()
+    path = joinpath(dirname(dirname(@__FILE__)), "BioFmtSpecimens")
+    if !isdir(path)
+        run(`git clone --depth 1 https://github.com/BioJulia/BioFmtSpecimens.git $(path)`)
+    end
+end
```

Comment:
See [a comment](https://github.com/BioJulia/GenomicFeatures.jl/commit/1ff45f3b345a854352409057e72332c6e1a75016#commitcomment-22601227) from tkelman. It may be better to check out a specific commit after cloning the repository. Also, returning the path will be useful. For example,
```julia
function get_bio_fmt_specimens(commit)
    path = joinpath(dirname(dirname(@__FILE__)), "BioFmtSpecimens")
    if !isdir(path)
        run(`git clone --depth 1 https://github.com/BioJulia/BioFmtSpecimens.git $(path)`)
    end
    run(`git checkout $(commit)`)
    return path
 end
```